plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -91,6 +93,8 @@
 
 public final class StandardColumnMappings
 {
+    private static final int MAX_LOCALDATETIME_PRECISION = 9;
```

Comment:
nit: ..._LOCAL_DATE_TIME_... since it's NotSingleWord

Reply:
Done!